src/components/common/CommonInput.js

```diff
@@ -0,0 +1,18 @@
+import { string } from "prop-types";
+
+import "./common-input.scss";
+
+const CommonInput = ({ labelText, inputType, placeHolderText }) => (
+  <label>
+    {labelText}
+    <input type={inputType} placeholder={placeHolderText} />
+  </label>
+);
+
+CommonInput.propTypes = {
+  labelText: string,
+  inputType: string,
+  placeHolderText: string,
+};
```

Comment:
Why are you using prop types instead of Typescript? Have you thought of using typescript instead?

Reply:
I was told not to use typescript because this way is simpler to learn. WDYT?

Reply:
Yes, I agree it's easier to use javascript instead of Typescript, In your current situation if I were you, I would keep using javascript but I didn't want to not mention this. It's a suggestion for the future when you feel more comfortable developing in React.